test/integration/security/access-sanity.js

```diff
@@ -106,6 +107,85 @@ describe(
       });
     });
 
+    context.skip('testing persisting on permissions', function() {
```

Comment:
Why is this being skipped?

Reply:
This is now being tested in /integration/security/advanced-event-security. Context block removed.